exist-core/src/main/resources/org/exist/test/runner/xquery-test-runner.xq

```diff
@@ -24,6 +24,8 @@ xquery version "3.1";
 import module namespace test = "http://exist-db.org/xquery/xqsuite"
     at "resource:org/exist/xquery/lib/xqsuite/xqsuite.xql";
 
+import module namespace inspect = "http://exist-db.org/xquery/inspection";
```

Comment:
This change is a clear hint that this constitutes a breaking change.
Library and main modules that use one of the functions in the inspect namespace will no longer function until they are imported.